web-portal/components/SubmissionForm.vue

```diff
@@ -658,17 +677,7 @@
       },
 
       suggestedTags: function () {
-        return [
```

Comment:
I'm confused by this? I thought we would still want to offer auto-complete tags. Rather than using these tags for auto-complete, I thought the idea was we actually populate the form with them.

I guess there's a language challenge here :-) 

To me these are a different kind of "suggested", maybe it's suggested tags vs predicted tags, or autoCompleteTags vs suggestedTags

Reply:
I just moved this list to the `TagSuggestionService` rather than have them hard-coded here.

Originally we were going to replace this hard-coded list of "suggested" (maybe "recommended" is more descriptive now?) with a call to the server to get the top X most frequently-used tags, and maybe filter that as you typed in the tag field, but never got around to it.

Reply:
Would it be more clear if it was named something like `$tagSuggestionService.getBaseTagSet()`?

Reply:
oh I see, mostly I was just confused :-)

yeah that works